clustermesh/certs.go

```diff
@@ -154,10 +154,22 @@ func (k *K8sClusterMesh) deleteCertificates(ctx context.Context) error {
 func (k *K8sClusterMesh) installCertificates(ctx context.Context) error {
 	err := k.certManager.LoadCAFromK8s(ctx)
 	if err != nil {
-		k.Log("❌ Cilium CA not found: %s", err)
-		return err
+		if !k.params.CreateCA {
+			k.Log("❌ Cilium CA not found: %s", err)
+			return err
+		}
+
+		k.Log("🔑 Generating CA...")
+		if err := k.certManager.GenerateCA(); err != nil {
+			return fmt.Errorf("unable to generate ca: %w", err)
```

Comment:
Spelling nit (`CA` is used in other log messages here):

```suggestion
			return fmt.Errorf("unable to generate CA: %w", err)
```